README.md

```diff
@@ -492,6 +495,52 @@ status:
     status: 'True'
 ```
 
+## Custom Binding Paths
+
+There are scenarios where the application resource is not PodSpec-able with custom binding paths for env, volumes, and volumeMounts.  This extension allows a `ServiceBinding` resource to specify custom binding paths for the application resource.  The custom binding paths **MUST** be one or more of env, volumes, and volumeMounts.  In the absence of any of these items, that particular projection **MUST NOT** be performed.  The `ServiceBinding` author **MUST** set the `service.binding/path` annotation to `Custom`.  The binding path value **MUST** be a [JSONPath][jp] location in the application resource.  The binding paths **MUST** be specified in a ConfigMap, and `.spec.application.bindingPaths` in `ServiceBinding` resource **MUST** refer to it.
+
+[jp]: https://kubernetes.io/docs/reference/kubectl/jsonpath/
+
+### Custom Binding Paths Example ConfigMap
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: example-binding-paths
+data:
```

Comment:
How would you express the default behavior in these semantics? I'm not sure mapping into specific containers can be expressed in JSONPath

Reply:
I have changed the approach in the modified text. In the new approach, there is a provision to bind the `containers` list.